terra-jupyter-gatk-ovtf/gatk-ovtf.patch

```diff
@@ -0,0 +1,118 @@
+From a22f79681d517ba2c6a7677638aab96c09a5aace Mon Sep 17 00:00:00 2001
```

Comment:
What is this `patch` used for, and is there a reason it needs to be checked in?

Reply:
Hi, below are the changes included in the patch:
- Enabling OpenVINO integration with TensorFlow for CNNScoreVariants inference
- Freezing the Keras model for inference to trim training related layers. (Based on: https://medium.com/@sebastingarcaacosta/how-to-export-a-tensorflow-2-x-keras-model-to-a-frozen-and-optimized-graph-39740846d9eb)
- Additional changes for TF 2.x compatibility

Reply:
Thank you for reviewing @rtitle.
The patch is used to update two python files `vqsr_cnn/inference.py` and `vqsr_cnn/vqsr_cnn/models.py`.

- These minor changes will help in additional performance improvement. 
- It needs to be checked as it will be used when building the docker image. Inside the docker image, this patch will be copied into it and will be used to update the GATK code.